k8sclient/k8sclient.go

```diff
@@ -621,13 +621,15 @@ func GetDefaultNetworks(k8sArgs *types.K8sArgs, conf *types.NetConf, kubeClient
 	delegate.MasterPlugin = true
 	delegates = append(delegates, delegate)
 
-	//need to revisit
-	for _, netname := range conf.DefaultNetworks {
-		delegate, err := getNetDelegate(kubeClient, netname, conf.ConfDir)
-		if err != nil {
-			return err
+	// Pod in kube-system namespace does not have default network for now.
+	if string(k8sArgs.K8S_POD_NAMESPACE) != "kube-system" {
```

Comment:
Do we want to make the special namespace list configurable instead of hardcode?

Reply:
It might be added in future feature, but for now we don't have use-case hence not a plan yet.

Reply:
Do you have any use-case for that?

Reply:
After reconsideration, I am even not sure why the 'kube-system' should be excluded from the 'defaultNetworks'

I was thinking that there are also some other namespaces may have pods running which does not need the secondary interface. But actually, both the kube-system and the namespace which contains the system services (eg, openshift-sdn) will have pod running with `hostNetwork: true`, which will not invoke the multus-cni already. 

AFAIK, we do not have any pod running in kube-system with container network, so do we really need this condition?

Reply:
The codeDNS will be up only after you apply a cni like calico or flannel before that it will be in pending state.

Can we give the multus `hostNetwork: true` and allow him to run if there is no cni install.
```yaml
      tolerations:
      - operator: Exists
        effect: NoSchedule
```
Then you can apply multus and the clusterNetwork CRD before the real cni.

Reply:
I'd like to keep to install multus without daemonset so I'd like to keep.